internal/runtimehandlerhooks/high_performance_hooks_linux.go

```diff
@@ -61,6 +61,49 @@ const (
 	SharedCPUsEnvVar     = "OPENSHIFT_SHARED_CPUS"
 )
 
+// ServiceManager interface for managing system services.
```

Comment:
Added new interfaces so that we can mock service state & restarts and command looksups and runs.